azure-pipelines.release.web-components.yml

```diff
@@ -0,0 +1,86 @@
+pr: none
+trigger: none
+
+# Customize build number to include major version
+# Example: web-components_20201022.1
+name: 'web-components_$(Date:yyyyMMdd)$(Rev:.r)'
+
+variables:
+  - group: 'Github and NPM secrets'
+  - template: .devops/templates/variables.yml
+    parameters:
+      skipComponentGovernanceDetection: false
+  - name: release.web_components # Used to scope beachball to release only vnext packages
```

Comment:
```suggestion
  - name: release.web_components # Used to scope beachball to release only web component packages
```

Reply:
Since we're technically only releasing v-next packages still, I'm going to suggest we not fix this as part of this PR, but we can clarify the language if need be.